python/ThirteenTeV/DisplacedSUSY_stopToBottom_M_300_100mm_TuneCUETP8M1_13TeV_pythia8_cff.py

```diff
@@ -0,0 +1,857 @@
+     0.00000000E+00    2     2000005        -5   # BR(~chi_40 -> ~b_2      bb)
+     0.00000000E+00    2    -2000005         5   # BR(~chi_40 -> ~b_2*     b )
+     9.64835418E-03    2     1000011       -11   # BR(~chi_40 -> ~e_L-     e+)
+     9.64835418E-03    2    -1000011        11   # BR(~chi_40 -> ~e_L+     e-)
+     3.75684470E-03    2     2000011       -11   # BR(~chi_40 -> ~e_R-     e+)
+     3.75684470E-03    2    -2000011        11   # BR(~chi_40 -> ~e_R+     e-)
+     9.64835418E-03    2     1000013       -13   # BR(~chi_40 -> ~mu_L-    mu+)
+     9.64835418E-03    2    -1000013        13   # BR(~chi_40 -> ~mu_L+    mu-)
+     3.75684470E-03    2     2000013       -13   # BR(~chi_40 -> ~mu_R-    mu+)
+     3.75684470E-03    2    -2000013        13   # BR(~chi_40 -> ~mu_R+    mu-)
+     2.68215241E-03    2     1000015       -15   # BR(~chi_40 -> ~tau_1-   tau+)
+     2.68215241E-03    2    -1000015        15   # BR(~chi_40 -> ~tau_1+   tau-)
+     1.62289809E-02    2     2000015       -15   # BR(~chi_40 -> ~tau_2-   tau+)
+     1.62289809E-02    2    -2000015        15   # BR(~chi_40 -> ~tau_2+   tau-)
+     2.53796547E-02    2     1000012       -12   # BR(~chi_40 -> ~nu_eL    nu_eb)
+     2.53796547E-02    2    -1000012        12   # BR(~chi_40 -> ~nu_eL*   nu_e )
+     2.53796547E-02    2     1000014       -14   # BR(~chi_40 -> ~nu_muL   nu_mub)
+     2.53796547E-02    2    -1000014        14   # BR(~chi_40 -> ~nu_muL*  nu_mu )
+     2.54724352E-02    2     1000016       -16   # BR(~chi_40 -> ~nu_tau1  nu_taub)
+     2.54724352E-02    2    -1000016        16   # BR(~chi_40 -> ~nu_tau1* nu_tau )
+"""
+
+import FWCore.ParameterSet.Config as cms
+
+from Configuration.Generator.Pythia8CommonSettings_cfi import *
+from Configuration.Generator.Pythia8CUEP8M1Settings_cfi import *
+
+generator = cms.EDFilter("Pythia8GeneratorFilter",
+    pythiaPylistVerbosity = cms.untracked.int32(0),
+    filterEfficiency = cms.untracked.double(1),
+    pythiaHepMCVerbosity = cms.untracked.bool(False),
+    comEnergy = cms.double(COM_ENERGY),
+    crossSection = cms.untracked.double(CROSS_SECTION),
+    maxEventsToPrint = cms.untracked.int32(0),
+    SLHATableForPythia8 = cms.string('%s' % SLHA_TABLE),
+    PythiaParameters = cms.PSet(
+        pythia8CommonSettingsBlock,
+        pythia8CUEP8M1SettingsBlock,
+        processParameters = cms.vstring(
+            'SUSY:all = off',
+            'SUSY:gg2squarkantisquark  = on',
+            'SUSY:qqbar2squarkantisquark= on',
+            'RHadrons:allow  = on',
+            'RHadrons:allowDecay = on',
+            'RHadrons:setMasses = on',
+            '1000006:tau0 = %.1f' % CTAU_POINT,
+        ),
+        parameterSets = cms.vstring(
+            'pythia8CommonSettings',
+            'pythia8CUEP8M1Settings',
+            'processParameters'
+        )
+    )
+)
+
+dirhadrongenfilter = cms.EDFilter("MCParticlePairFilter",
+    #Status = cms.untracked.vint32(1, 1),
```

Comment:
Ok, sorry never mind, looking at the code now.  Understand that the warning is just because you left out the "status" vector from the config.  (Which means that it is filled with 0, which means that the status is not used).

So ok the code is ugly (and also takes untracked parameters for the filter settings!) but fine.